src/app/pages/OnDemandAudioPage/OnDemandAudioPage.tsx

```diff
@@ -157,106 +126,84 @@ const OnDemandAudioPage = ({
         {...metadataImageProps}
         hasAmpPage={false}
       />
-      {/* @ts-expect-error: Legacy grid expects `children` to be passed as props. However, due to coding best practices, we must nest children between the opening and closing tags */}
-      <GelPageGrid
-        as="main"
-        role="main"
-        columns={getGroups(6, 6, 6, 6, 8, 20)}
-        enableGelGutters
-      >
-        {/* @ts-expect-error: Legacy grid expects `children` to be passed as props. However, due to coding best practices, we must nest children between the opening and closing tags */}
-        <Grid
-          item
-          startOffset={getGroups(1, 1, 1, 1, 2, 5)}
-          columns={getGroups(6, 6, 6, 6, 6, 12)}
-          margins={getGroups(true, true, true, true, false, false)}
-        >
-          {/* @ts-expect-error: Legacy grid expects `children` to be passed as props. However, due to coding best practices, we must nest children between the opening and closing tags */}
-          <GelPageGrid
-            dir={oppDir}
-            columns={getGroups(6, 6, 6, 6, 6, 6)}
-            enableGelGutters
-            css={styles.wrapper}
-          >
-            {/* @ts-expect-error: Legacy grid expects `children` to be passed as props. However, due to coding best practices, we must nest children between the opening and closing tags */}
-            <Grid
-              item
-              columns={getGroups(6, 6, 4, 4, 4, 4)}
-              parentColumns={getGroups(6, 6, 6, 6, 6, 6)}
-              parentEnableGelGutters
-              css={styles.paragraph}
-            >
-              <StyledRadioHeadingContainer
-                // @ts-expect-error idAttr is a string
-                idAttr={idAttr}
+      <div css={styles.grid}>
+        <div css={styles.contentWrapper}>
+          <main role="main">
+            <div css={styles.flexWrapper}>
+              <div css={styles.text}>
+                <StyledRadioHeadingContainer
+                  // @ts-expect-error idAttr is a string
+                  idAttr={idAttr}
```

Comment:
Does this work? (I know it was an existing issue but would be nice if we could fix while we're in here!)
```suggestion
                  idAttr={idAttr as string}
```

Reply:
Have made some updates to OnDemandHeading that seem to fix this and not break anything else. (Have also added a test).